CRM/Core/DAO/AllCoreTables.php

```diff
@@ -280,7 +280,7 @@ public static function getClasses() {
    * Get the classname for the table.
    *
    * @param string $tableName
-   * @return string
+   * @return CRM_Core_DAO|NULL
    */
```

Comment:
The return *is* a string?

Reply:
For convenience of IDE auto-completion, we kinda hack the `@return` annotationwhen it's returning a string which is also a class name. In php strings can be used interchangeably with classes, go figure.